app/src/main/java/com/eveningoutpost/dexdrip/Services/DexShareCollectionService.java

```diff
@@ -289,6 +289,11 @@ public void call(EGVRecord[] egvRecords) {
                                         if (egvRecords != null) {
                                             Log.d(TAG, "Made the full round trip, got " + egvRecords.length + " EVG Records");
                                             BgReading.create(egvRecords, addativeSystemTimeOffset, getApplicationContext());
+                                            {
```

Comment:
My mistake.
Looking this at github gives a very bad impression of what is actually going on....
Had to clone the repository to really see it.